Tailviewer.AcceptanceTests/BusinessLogic/LogFiles/TextLogFileTest.cs

```diff
@@ -126,16 +124,16 @@ public void TestDoesNotExist()
 		{
 			_file = new TextLogFile(_scheduler, _fname);
 			_scheduler.RunOnce();
-			_file.GetValue(LogFileProperties.EmptyReason).Should().Be(ErrorFlags.None);
-			_file.GetValue(LogFileProperties.Created).Should().NotBe(DateTime.MinValue);
```

Comment:
This merge doesn't look right, GetValue(...) is the new way to retrieve data, please don't revert that.